src/main/java/com/epam/emotionalhelp/service/impl/QuizResultServiceImpl.java

```diff
@@ -14,9 +14,9 @@
 @RequiredArgsConstructor
 public class QuizResultServiceImpl implements QuizResultService {
     @Override
-    public Map<Long, Integer> calculateResult(List<EmotionDto> emotions) {
+    public Map<Long, Double> calculateResult(List<EmotionDto> emotions) {
         return emotions.stream()
                 .collect(Collectors.groupingBy(EmotionDto::getEmotionId, LinkedHashMap::new,
-                        Collectors.summingInt(EmotionDto::getValue)));
+                        Collectors.averagingInt(emotionDto -> emotionDto.getValue() * 20)));
```

Comment:
What is a magic number 20 - constant?

Reply:
Hi @maximelyashevich . We multiply interval [0;5] by koeficient 20 and get a new one [0;100], which represents percents. 0-5 is a value of user emotion we get from FE

Reply:
Thanks for the provided details - let's add a constant in this class with this value.